deployment/deployment_test.go

```diff
@@ -162,8 +162,10 @@ func TestCreate(t *testing.T) {
 	role := "test-role"
 	executor := "CeleryExecutor"
 	airflowVersion := "1.10.5"
+	dagDeploymentType := "image"
+	nfsLocation := ""
 	buf := new(bytes.Buffer)
-	err := Create(label, ws, releaseName, role, executor, airflowVersion, api, buf)
+	err := Create(label, ws, releaseName, role, executor, airflowVersion, dagDeploymentType, nfsLocation, api, buf)
```

Comment:
😰 This is a lot of params for a function.  Maybe we can mark this as a todo to clean up later?

Reply:
yeah we have to switch to struct instead, okey will do.